aws-datastore/src/androidTest/java/com/amplifyframework/datastore/HybridCloudSyncInstrumentationTest.java

```diff
@@ -116,11 +116,12 @@ public void setup() throws AmplifyException {
         api = SynchronousApi.delegatingTo(apiCategory);
         appSync = SynchronousAppSync.using(AppSyncClient.via(apiCategory));
 
+        schemaProvider = SchemaLoader.loadFromAssetsDirectory("schemas/commentsblog");
```

Comment:
The key change in this PR is that the schema are being read from JSON. The JSON files no longer say `com.path.to.some.generated.java.model.JavaPojo`. (Previously, the test had passed, with those queues.)

Now, the stated type for a model is a serialized model.